Restarting the CI build causes linter warnings

I created a fork of this repository and pushed an empty commit to trigger a CI build based off of the master branch, and I got this failure:

https://travis-ci.org/nicolasmccurdy/karma-ie-launcher/jobs/126575282
```
$ grunt
Running "eslint:target" (eslint) task

/home/travis/build/nicolasmccurdy/karma-ie-launcher/test/launcher.spec.js
  151:9   error  Identifier 'child_processCmd' is not in camel case   camelcase
  155:13  error  Identifier 'child_processMock' is not in camel case  camelcase
  156:9   error  Identifier 'child_processMock' is not in camel case  camelcase
  158:13  error  Identifier 'child_processCmd' is not in camel case   camelcase

✖ 4 problems (4 errors, 0 warnings)

Warning: Task "eslint:target" failed. Use --force to continue.

Aborted due to warnings.
```

It's possible that an updated dependency version had a breaking change, or that the build is configured differently for forks.

--- a/test/launcher.spec.js
+++ b/test/launcher.spec.js
@@ -148,20 +148,20 @@ describe('launcher', function () {
   })
 
   describe('_onProcessExit', function () {
-    var child_processCmd, onProcessExit
+    var childProcessCmd, onProcessExit
 
     beforeEach(function () {
       onProcessExit = function () {
-        var child_processMock
-        child_processMock = {
+        var childProcessMock
+        childProcessMock = {
           exec: function (cmd, cb) {
-            child_processCmd = cmd
+            childProcessCmd = cmd
             cb()
           }
         }
 
         IELauncher = mocks.loadFile(path.join(__dirname, '/../index'), {
-          child_process: child_processMock
+          child_process: childProcessMock
         }).module.exports
         injector = new di.Injector([module, IELauncher])
         launcher = injector.get('launcher:IE')
@@ -174,7 +174,7 @@ describe('launcher', function () {
 
     it('should call wmic with process ID', function (done) {
       onProcessExit()
-      expect(child_processCmd).to.equal(
+      expect(childProcessCmd).to.equal(
         'wmic.exe Path win32_Process where ' +
         '"Name=\'iexplore.exe\' and CommandLine Like \'%SCODEF:10%\'" call Terminate'
       )

